.github/workflows/build.yml

```diff
@@ -148,19 +152,23 @@ jobs:
     name: Linux (x86_64)
     needs: [version]
     runs-on: ubuntu-latest
-    env: 
-      BUILDER: docker run -v .:/build -w /build ghcr.io/lite-xl/lite-xl-build-box-manylinux:v3.0.2 -- /entrypoint.sh
     steps:
       - uses: actions/checkout@v4
         
       - name: Build
-        run: $BUILDER scripts/build.sh --addons --debug --forcefallback --portable -b build
+        uses: docker://ghcr.io/lite-xl/lite-xl-build-box-manylinux:v3
```

Comment:
Might be a good idea to have this defined once somewhere at the top.

Reply:
The container "url", so that you don't need to copypaste it around.

Reply:
It's used in 2 lines in the whole file, and they're like 5 lines away. I don't think it warrants declaring an environment variable or something.

Reply:
Fair enough.